precommit/src/main/shell/plugins.d/htmlout.sh

```diff
@@ -283,3 +297,39 @@ function htmlout_finalreport
 
   printf "<p>This message was automatically generated.</p>" >> "${commentfile}"
 }
+
+
+## @description  Write out an HTML version of the final report to a file
+## @audience     private
+## @stability    evolving
+## @replaceable  no
+## @param        runresult
+function htmlout_finalreport
+{
+  declare result=$1
+
+  rm "${commentfile}" 2>/dev/null
```

Comment:
Why `rm` with error output ignored instead of something like `rm -f` ?

Reply:
It might not exist due to a previous failure elsewhere.  That's ok and we don't want the error to go to console.